spec/config/initializers/ab_tests_spec.rb

```diff
@@ -1,47 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe AbTests do
-  def reload_ab_test_initializer!
-    # undefine the AB tests instances so we can re-initialize them with different config values
-    AbTests.constants.each do |const_name|
```

Comment:
These tests are difficult to refactor as frozen constants, though it is in theory possible. The tests are mostly tests covering the behavior already tested in `spec/lib/ab_test_bucket_spec.rb`